CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Document picker/DocumentPickerCoordinator.swift

```diff
@@ -197,6 +196,24 @@ public final class DocumentPickerCoordinator: NSObject {
 
 // MARK: - Fileprivate methods
 
+/// A UIDocumentPickerViewController that sets navigation bar buttons' tint color to a correct one, instead of using the accent color specified by app
+private class GiniDocumentPickerViewController: UIDocumentPickerViewController {
+    var initialTintColor: UIColor? = navBarInstance.tintColor
+
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        Self.navBarInstance.tintColor = .GiniCapture.accent1
+    }
+    override func viewDidDisappear(_ animated: Bool) {
+        super.viewDidDisappear(animated)
+        // make sure to revert the tint color, otherwise it will affect document pickers/browsers from the host app
+        Self.navBarInstance.tintColor = initialTintColor
```

Comment:
[Suggestion]: we need to check if initialTintColor exist

Reply:
This just returns the color to the initial state. Setting that to nil will just make the system use default color.